Files/UserControls/FilePreviews/MediaPreview.xaml

```diff
@@ -12,6 +12,7 @@
 
     <Grid>
         <MediaPlayerElement
+            x:Name="PlayerContext"
```

Comment:
Can you use x:Bind to set the volume instead of handling it in code behind?

Reply:
it doesn't have to be a dependency property with x:Bind

Reply:
and how would you bind it to MediaPlayerElement?
the volume property is under 
`MediaPlayerElement.MediaPlayer.MediaVolume`

Reply:
Ah, sorry my bad :)